numba/cuda/target.py

```diff
@@ -88,7 +90,7 @@ def init(self):
     def load_additional_registries(self):
         # side effect of import needed for numba.cpython.*, the builtins
         # registry is updated at import time.
-        from numba.cpython import numbers, tupleobj, slicing # noqa: F401
+        from numba.cpython import numbers, tupleobj, slicing, enumimpl # noqa: F401, E501
```

Comment:
Just to keep the lines within the limit and avoid an extra `noqa`, something along the lines of:

```suggestion
        from numba.cpython import numbers, tupleobj, slicing # noqa: F401
        from numba.cpython import enumimpl # noqa: F401
```

(or it could be added to the following line with `rangeobj` and `iterators`)